src/migratesettings.h

```diff
@@ -3,7 +3,7 @@
 
 #include "settings.h"
 
-void migrate_settings()
+void migrate_settings(void)
```

Comment:
Don't. [There is no point for this in C++.](https://softwareengineering.stackexchange.com/questions/286490/what-is-the-difference-between-function-and-functionvoid)

Reply:
oh sorry I was just trying to get it to reload and it wouldnt without a change and i couldnt push bc i jave like 100kbps internet rn

Reply:
oh sorry I was just trying to get it to reload and it wouldnt without a change and i couldnt push bc i jave like 100kbps internet rn

Reply:
oh sorry I was just trying to get it to reload and it wouldnt without a change and i couldnt push bc i jave like 100kbps internet rn